corehq/apps/translations/app_translations/upload_app.py

```diff
@@ -29,8 +29,9 @@
     is_module_sheet,
     is_modules_and_forms_sheet,
     is_single_sheet,
-    is_single_sheet_workbook,
+    is_single_sheet_workbook
 )
+from corehq.apps.users.extension_points import update_audio_path_if_required, extract_older_rows
```

Comment:
Why is `extension_points` in `users`?  Wouldn't it make more sense in this same app (`translations`)?

Reply:
Thanks for pointing that out @esoergel. It definitely should be in translations. I might have created it mistakingly there and then never gave it a thought.
Would update it.